.github/batch_changes/migrations/001-connectors-workflow-and-precommit-updates.yaml

```
@@ -0,0 +1,82 @@
+name: 001-connectors-workflow-and-precommit-updates
+description: Add workflow updates and standardized pre-commit changes and configurations to connectors for frictionless connector contributions [PAPP-35399]
+
+on:
+  - repositoriesMatchingQuery: repo:^github\.com/splunk-soar-connectors/ -repo:^github\.com/splunk-soar-connectors/\.github -repo:^github\.com/splunk-soar-connectors/frictionless_connectors_test
+
+steps:
+  # Step 1: Workflow files
+  - container: alpine:3
+    env:
+      REPO: phantomcyber/dev-cicd-tools
+      BRANCH: main
+    run: |
+      apk add --no-cache curl
+
+      # Remove existing workflow files
+      rm -rf .github/workflows/*
+      mkdir -p .github/workflows
+
+      # Download workflow files
+      curl -LsSfo .github/workflows/call-push.yml https://raw.githubusercontent.com/${REPO}/${BRANCH}/templates/call-push-template.yml
+      curl -LsSfo .github/workflows/call-publish.yml https://raw.githubusercontent.com/${REPO}/${BRANCH}/templates/call-publish-template.yml
+
+  # Step 2: Add new pre-commit configurations and apply changes
+  - container: quay.io/pypa/manylinux_2_28_x86_64 # Needed for package app dependencies (and other script entries too)
+    env:
+      REPO: phantomcyber/dev-cicd-tools
+      BRANCH: main
+    run: |
+      # Dependencies
+      yum install -y \
+        curl \
+        git \
+        python39 \
+        python39-pip \
+        jq
+
+      # 3.9 default
+      alternatives --set python3 /usr/bin/python3.9
+      
+      python3 -m pip install --upgrade "pre-commit>=4.1.0"
+      pre-commit --version
+
+      # Remove tox file if exists (no longer needed)
+      rm -f tox.ini
+
+      # Download our template files
+      curl -LsSfO https://raw.githubusercontent.com/${REPO}/${BRANCH}/templates/pre-commit-hooks-template.yaml
+      curl -LsSfO https://raw.githubusercontent.com/${REPO}/${BRANCH}/templates/pyproject.toml
+
+      # Rename templates
+      mv pre-commit-hooks-template.yaml .pre-commit-config.yaml
+
+      pre-commit install
+
+      # Run pre-commit up to 3 times (might need multiple iterations till everything is updated and clean)
```

Did we want to skip any hooks? Maybe
```suggestion
      # Run pre-commit up to 3 times (might need multiple iterations till everything is updated and clean)
      SKIP="package_app_dependencies"
```
for example

---

Yeah we can skip that if we desire, updated.